pkg/pillar/types/zedroutertypes.go

```diff
@@ -2889,3 +2889,24 @@ func (status OnboardingStatus) LogDelete(logBase *base.LogObject) {
 func (status OnboardingStatus) LogKey() string {
 	return string(base.OnboardingStatusLogType) + "-" + status.Key()
 }
+
+// AppInstMetaDataType - types of app meta data
+type AppInstMetaDataType uint8
+
+// enum app metadata type
+const (
+	AppInstMetaDataTypeNone AppInstMetaDataType = iota // enum for app inst metadata type
+	AppInstMetaDataTypeKubeConfig
+)
+
+// AppInstMetaData : App Instance Metadata
+type AppInstMetaData struct {
+	AppInstUUID string
```

Comment:
This should be of type uuid.UUID and not string so we can have some type checking.

Reply:
Resolved